util/src/windows/scoped_handle.cc

```diff
@@ -0,0 +1,17 @@
+#include <leatherman/util/windows/scoped_handle.hpp>
+#include <windows.h>
+
+using namespace std;
+
+namespace leatherman { namespace util { namespace windows {
+
+    scoped_handle::scoped_handle(HANDLE h) :
+        scoped_resource(h, CloseHandle)
```

Comment:
Ok, I can look at that. I figured CloseHandle was smart enough to essentially do that check.